pom.xml

```diff
@@ -28,6 +28,34 @@
     </repository>
   </repositories>
   <dependencies>
+    <dependency>
```

Comment:
I think you should be able to remove all dependencies except the one for JUnit as there is no mocking in these tests and I'd suggest removing the one which uses reflection.